internal/cmd/certgen.go

```diff
@@ -71,6 +80,9 @@ func certGen(ctx context.Context, logOut io.Writer, local bool) error {
 		if err = outputCertsForKubernetes(ctx, cli, cfg, overwriteControlPlaneCerts, certs); err != nil {
 			return fmt.Errorf("failed to output certificates: %w", err)
 		}
+		if err = patchTopologyInjectorWebhook(ctx, cli, cfg, certs.CACertificate); err != nil {
```

Comment:
lets track this as a follow up issue - there's a case where the user brings their own certs and drops it as a Secret, in that case we'll need to read the Secret, copy the ca cert from it and use it

Reply:
Is that supported today in the normal cert gen flow? Can certainly be supported though and would just be whatever caBundle is determined above this code.

Reply:
its supported today, since we are so close to the rc release, dont want to block on this, we can get this into v1.5, the subset of users using BYO certs + wanting zone aware routing, may need to do some gymnastics in v1.4, but it will be temporary

Reply:
https://gateway.envoyproxy.io/docs/install/custom-cert/